az-hadoop-jobtype-plugin/src/test/java/azkaban/jobtype/TestHadoopJobUtilsConstructHadoopTags.java

```diff
@@ -0,0 +1,48 @@
+package azkaban.jobtype;
+
+import azkaban.utils.Props;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import junit.framework.Assert;
```

Comment:
Thanks for improving tests!

Please use assertj for new tests. See the contributor's guide.

Reply:
Thanks for the pointer @HappyRay , updated the PR